packages/itwin/breakdown-trees/src/Views/FunctionalityProviders/CombinedTreeNodeFunctionalityProvider.ts

```diff
@@ -37,7 +37,7 @@ export class CombinedTreeNodeFunctionalityProvider extends TreeNodeFunctionality
         return this._groupNodeFunctionalityProvider.performAction([node], treeModel);
     } else if (NodeKey.isInstancesNodeKey(elementKey)){
       const classHierarchyArray = await IModelReadRpcInterface.getClient().getClassHierarchy(this._treeDataProvider.imodel.getRpcProps(), elementKey.instanceKeys[0].className);
-      for (let className of classHierarchyArray){
+      for (const className of classHierarchyArray){
```

Comment:
here and in other places of this file a whitespace is missing `for (const i of x){`